### ISSUE TITLE
HGR - Heydrich playthrough list of bugs

### ISSUE THREAD

**Issue by cheeems02** [TNODevs/TNO-Bug-Tracker#3477]
### Operating System

Windows 10

### HOI4 version

1.14.7

### TNO version

v1.6.0

### Do you have other mods in use?

none

### Were you using Steam?

Yes

### Which major expansions do you NOT have?

TOA

### Describe the bug

SS civil war bugs: 
1. When you save the prison you don't get Speidel for some reason, I didn't have him my entire game.
2. Feldmeier didn't take over Netherlands, I had Rochemont entire time.
3. Vituška focuses didn't even exist anywhere in Heydrich tree, which was unexpected considering his path was created with idea of him winning in Ostland.
4. Cornwall stuff doesn't exist, yet focuses do 
5. If you go German SS path you don't get Gille in Kommandostab which you should, you get that Himmler's guy Hausser and it's hard(er) to sway him to your side
6. Did focus to kill Hausser, event just changed the ruling party, probably relic of the time when purple ideology existed as a separate thing, Hausser was still alive and ruling the statelet.
7. When I annexed it, I got Hausser as field marshal, pretty sure he shouldn't join
8. Burgundy annexed it's offsprings before I was able to even do the first focus, had war with Degrelle tho.
9. Schörner's coup doesn't set Remer's statelet to nazi party at all and doesn't make him country leader. After you annex his statelet he doesn't join you, he doesn't even exist there, I didn't have him as field marshal or general or anything
10. Did Josias as first focus and for some reason he actually joined my faction
11. Got the event of Rommel retiring, pretty sure he should be in prison or dead when you conquer Berlin and imprison Speidel's guys. 
12. To ally with Poland you need to have at least negotiated with Children of Spartakus which are for some reason harder to convince than Poland. To ally Children of Spartakus, to even access the path to ally with them, you need to kill Eichmann's statelet south of East Prussia, encircled by (former) Ostland, Poland and the said Children of Spartakus. To reach them you would at least need to kill Poland. This all makes allying Poland completely impossible without cheating, which isn't the purpose of this.
13. Once Heydrich lost his shit and suicided himself I got CtD, tried reloading the latest save there and it always happened again and again.
14. 

### Steps to reproduce / Cause of the issue

1. Play Heydrich path
2. Go German SS
3. Complete almost entire focus tree for German SS before winning civil war
4. Deal with the above mentioned statelets in above mentioned ways
5. You will see all of the bugs I did

### Expected behaviour

basically the opposite of what happened to me in the describe the bug section

**Comment by Stormeagle449** [comment]
Fixed in current version.
